config/playground/helm/dev/helm-opensearch-dashboards.yaml

```diff
@@ -9,9 +9,9 @@ opensearchHosts: "https://opensearch-cluster-leader:9200"
 replicaCount: 2
 
 image:
-  repository: "opensearchproject/opensearch-dashboards"
+  repository: "opensearchplayground/opensearch-dashboards"
   # override image tag, which is .Chart.AppVersion by default
-  tag: ""
+  tag: "latest"
   pullPolicy: "IfNotPresent"
```

Comment:
latest, need to always pull

Reply:
agree, changed to use Always in [new commit](https://github.com/opensearch-project/dashboards-anywhere/pull/91/commits/afa1c7d57264e293228d81b98676ddfc36f9cac4)